code/modules/flufftext/Hallucination.dm

```diff
@@ -210,26 +211,115 @@ Gunshots/explosions/opening doors/less rare audio (done)
 	image_icon = 'icons/mob/animal.dmi'
 	image_state = "clown"
 
-/obj/effect/hallucination/simple/clown/New(loc,var/mob/living/carbon/T,duration)
+/obj/effect/hallucination/simple/clown/Initialize(mapload, var/mob/living/carbon/T,duration)
 	..(loc, T)
 	name = pick(clown_names)
 	QDEL_IN(src,duration)
 
 /obj/effect/hallucination/simple/clown/scary
 	image_state = "scary_clown"
 
+/obj/effect/hallucination/simple/borer
+	image_icon = 'icons/mob/animal.dmi'
+	image_state = "brainslug"
+
+/obj/effect/hallucination/borer
+	//A borer paralyzes you and crawls in your ear
+	var/obj/machinery/atmospherics/components/unary/vent_pump/pump = null
+	var/obj/effect/hallucination/simple/borer/borer = null
+
+/obj/effect/hallucination/borer/Initialize(mapload, var/mob/living/carbon/T)
+	..()
+	target = T
+	for(var/obj/machinery/atmospherics/components/unary/vent_pump/U in orange(7,target))
+		if(!U.welded)
+			pump = U
+			break
+	if(pump)
+		borer = new(pump.loc,target)
+		for(var/i=0, i<11, i++)
+			walk_to(borer, get_step(borer, get_cardinal_dir(borer, T)))
+			if(borer.Adjacent(T))
+				T << "<span class='userdanger'>You feel a creeping, horrible sense of dread come over you, freezing your limbs and setting your heart racing.</span>"
+				T.Stun(4)
+				sleep(50)
+				qdel(borer)
+				sleep(rand(60, 90))
+				T << "<span class='changeling'><i>Primary [rand(1000,9999)] states:</i> [pick("Hello","Hi","You're my slave now!","Don't try to get rid of me...")]</span>"
+				break
+			sleep(4)
+		if(!QDELETED(borer))
+			qdel(borer)
+	qdel(src)
+
+/obj/effect/hallucination/simple/bubblegum
+	name = "Bubblegum"
+	image_icon = 'icons/mob/lavaland/96x96megafauna.dmi'
+	image_state = "bubblegum"
+	px = -32
+
+/obj/effect/hallucination/oh_yeah
+	var/turf/closed/wall/wall
+	var/obj/effect/hallucination/simple/bubblegum/bubblegum = null
+	var/image/fakebroken
+	var/image/fakerune
+
+/obj/effect/hallucination/oh_yeah/Initialize(mapload, var/mob/living/carbon/T)
+	..()
+	target = T
+	for(var/turf/closed/wall/W in range(7,target))
+		wall = W
+		break
+	if(wall)
+		fakebroken = image('icons/turf/floors.dmi', wall, "plating", layer = TURF_LAYER)
+		var/turf/landing = get_turf(target)
+		var/turf/landing_image_turf = get_step(landing, SOUTHWEST) //the icon is 3x3
+		fakerune = image('icons/effects/96x96.dmi', landing_image_turf, "landing", layer = ABOVE_OPEN_TURF_LAYER)
+		fakebroken.override = TRUE
+		if(target.client)
+			target.client.images |= fakebroken
+			target.client.images |= fakerune
+		target << sound('sound/effects/meteorimpact.ogg', volume = 150)
+		bubblegum = new(wall, target)
+		sleep(10) //ominous wait
+		var/charged = FALSE //only get hit once
+		while(get_turf(bubblegum) != landing)
+			bubblegum.forceMove(get_step_towards(bubblegum, landing))
+			bubblegum.setDir(get_dir(bubblegum, landing))
+			target << sound('sound/effects/meteorimpact.ogg', volume = 150)
```

Comment:
Ah, good to know, i'll replace the sounds with that. Even better, since we'll get the distortion as well.